src/Orbits.jl

```diff
@@ -1,194 +1,385 @@
-- mtilde: (m1_i+m2_f)/(m1_i+m2_i) [unitless], here we have
-    - m1:    mass of non-exploding component
-    - m2i:   mass of exploding component
-    - m2f:   post explosion mass of exploding component
-- vkick_div_vrel: kick velocity in units of the pre-explosion relative velocity    [unitless]
-    - vrel = sqrt(G(m1_i + m2_i)/a)
-- theta: polar angle indicating kick direction       [rad]
-- phi:   azimuthal angle indicating kick direction   [rad]
+    RV_semiamplitude_K1(;m_1, m_2, P, e, i)
+
+Compute the amplitude of radial velocity variations given orbital parameters and masses
+
+# Arguments:
+- m_1:   mass of observed star             [Msun]
+- m_2:   mass of companion                 [Msun]
+- P:     orbital period                    [d]
+- e:     orbital eccentricity              [dimensionless]
+- i:     orbital inclination               [rad]
+
+# Output:
+- K1: amplitude of radial velocity variation of star 1  [km/s]
+"""
+function RV_semiamplitude_K1(;m_1, m_2, P, e, i)
+    return (m_2*sin(i))*cbrt(2*π*cgrav/(P*day_in_sec)*m_sun/(m_2+m_1)^2)/sqrt(1-e^2)/(km) #semi amplitude of RV variation in km/s
+end
+
+
+###########################################################
+### 
+### Post kick orbital properties, for circular pre-explosion orbits
+### 
+###########################################################
+
+"""
+    post_supernova_circular_orbit_a(;m_1i, m_2i, a_i, m_1f=-1, m_2f, vkick=0, θ=0, ϕ=0, vimp=0)
+
+Compute post-kick properties for a circular pre-explosion orbit using equations 
+from Tauris & Takens (1999), except that here we have M2 as the exploding star.
+
+# Arguments:
+- m_1i:  pre-explosion  mass of non-exploding component [Msun]           
+- m_2i:  pre-explosion  mass of exploding component     [Msun]       
+- a_i:   pre-explosion orbital separation               [cm]
+- m_1f:  post-explosion mass of non-exploding component [Msun]           
+- m_2f:  post-explosion mass of exploding component     [Msun]   
+- vkick: kick velocity                                  [km/s] 
+- θ:     polar kick angle (away from e_par)             [rad]
+- ϕ:     azimuthal kick angle (off of e_perp)           [rad]
+- vimp:  imparted kick velocity on companion            [km/s]     
     
-#Output:
-- afinal: post-explosion orbital separation          [cm]
-- efinal: post-explosion excentricity                [dimensionless]
+# Output:
+- a_f: post-explosion orbital separation                [cm]
+- e_f: post-explosion excentricity                      [dimensionless]
 """
-function post_kick_parameters_a(a,mtilde,vkick_div_vrel,theta,phi)
-    xi = (1+vkick_div_vrel^2+2*vkick_div_vrel*cos(theta))/(mtilde)
-    Q = xi - 1 - (vkick_div_vrel*sin(theta)*cos(phi))^2/(mtilde)
+function post_supernova_circular_orbit_a(;m_1i, m_2i, a_i, m_1f=-1, m_2f, vkick=0, θ=0, ϕ=0, vimp=0)
+    if m_1f == -1
+        m_1f = m_1i
+    end
+    mtilde = (m_1f+m_2f)/(m_1i+m_2i) 
+    v_rel = average_orbital_velocity(;m_1=m_1i, m_2=m_2i, a=a_i)
```

Comment:
here as well, you don't need the semicolon to specify the start of the kwargs when calling the function